hack/check-links.sh

```diff
@@ -4,8 +4,15 @@ set -e
 
 source ./hack/lib/common.sh
 
-header_text "Building the site and checking links"
+header_text "Building the website"
 docker volume create sdk-html
+trap_add "docker volume rm sdk-html" EXIT
 docker run --rm -v "$(pwd):/src" -v sdk-html:/src/website/public klakegg/hugo:0.73.0-ext-ubuntu -s website
-docker run --rm -v sdk-html:/target mtlynch/htmlproofer /target --empty-alt-ignore --http-status-ignore 429 --allow_hash_href
-docker volume rm sdk-html
+
+header_text "Checking links"
+# For config explanation: https://github.com/gjtorikian/html-proofer#special-cases-for-the-command-line
+docker run --rm -v sdk-html:/target klakegg/html-proofer:latest /target \
```

Comment:
@asmacdo I changed this image from the seemingly unmaintained `mtlynch/htmlproofer` to `klakegg/html-proofer`. That won't be a problem I assume?